integrations/gen_docs_integrations.py

```diff
@@ -117,6 +117,25 @@ def read_integrations_js(path_to_file):
         print("Exception", e)
 
 
+def create_img_link(integration, filename):
```

Comment:
why passing filename? it is always `integration['meta']['monitored_instance']['icon_filename']` and you pass integration!

Reply:
no it ain't, exporters and notifications don't have monitored instance